### PR TITLE
Adding post aws token example

### PR DESCRIPTION
**Pull request by NicolasFieldwire**

### CODE REVIEW

File: fieldwire.rb [hunk 1]
```diff
@@ -54,6 +55,23 @@ def patch(url, project_token, attributes)
   JSON.parse(response.body)
 end
 
+def postToAws(url, attributes, filename)
```
Comment (skalb) [inline fieldwire.rb]: post_to_aws

File: fieldwire.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/ruby
 
 require 'HTTParty'
+require 'rest-client'
```
Comment (skalb) [inline fieldwire.rb]: Do we need both HTTParty and RestClient? Can we just use RestClient?

**Review by skalb — CHANGES_REQUESTED** [review]

File: fieldwire.rb (hunk 1 above)
Comment (NicolasFieldwire) [reply fieldwire.rb]: Done

File: fieldwire.rb (hunk 2 above)
Comment (NicolasFieldwire) [reply fieldwire.rb]: Yes. Done.

File: Gemfile [hunk 3]
```diff
@@ -1,5 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.0.0'
+ruby '2.4.1'
 
-gem 'httparty'
\ No newline at end of file
+gem 'httparty', '0.14.0'
```
Comment (skalb) [inline Gemfile]: Can't we remove this dependency now?

**Review by skalb — CHANGES_REQUESTED** [review]

**Comment by NicolasFieldwire** [comment]
Sure.

**Review by skalb — APPROVED** [review]

**Pull request merged**